ush/create_diag_table_file.py

```diff
@@ -21,12 +20,12 @@
 
 
 def create_diag_table_file(run_dir):
-    """Creates a diagnostic table file for each cycle to be run
+    """Creates an FV3 diagnostic table (``diag_table``) file for each cycle to be run
 
     Args:
-        run_dir: run directory
+        run_dir (str): run directory
```

Comment:
Should the r in run be capitalized?

```suggestion
        run_dir (str): Run directory
```